src/components/InfinityModal/InfinityModalForm.css

```diff
@@ -0,0 +1,240 @@
+#infinity-mode-form {
```

Comment:
Maybe you could use css.modules instead of common css?

Reply:
true, I did this as Conan was set up due to time constraints.   it is up to you if you need them as modules before merge or can accept as is.   I am not familiar with css.modules so I may not get to that change for a while.